src/contentApi/get-apps-data.js

```diff
@@ -12,6 +14,12 @@ const getAppsData = () => {
     getFifiDataSchema(),
     getCostDataSchema(),
   ];
+  if (
+    window.insights.chrome.getEnvironment() === 'stage' &&
+    window.insights.chrome.isBeta()
+  ) {
+    data.push(getManagedServicesDataSchema());
+  }
```

Comment:
maybe this could be another condition/method as for permissions/entitlements. 🤔

Reply:
Nah its just temporary because the app will be released at the summit. It does not exist now.